### PR TITLE
fix(forms): apply border styling for form input fields in focus

### PR DESCRIPTION
**Pull request by ybaskaran**
**What is the problem this PR is trying to solve?**
Multiple borders for form input fields on focus

**What is the chosen solution to this problem?**
Apply border bottom styling for form input fields on focus

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/src/theme/_forms.scss [hunk 1]
```diff
@@ -82,7 +82,8 @@ form {
 		border-bottom: $form-border-bottom;
 
 		&:focus {
-			@include box-shadow($form-focus-shadow);
+			border-bottom: $form-focus-border-bottom;
+			box-shadow: none;
```
Comment (frassinier) [inline packages/theme/src/theme/_forms.scss]: Why do you have to fix `box-shadow: none` if none of them use it?

Comment (ybaskaran) [reply packages/theme/src/theme/_forms.scss]: fixed, reverting border-bottom styles with box-shadow as it introduced alignment issues with input fields

File: packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.scss [hunk 2]
```diff
@@ -12,8 +12,7 @@ $tf-typeahead-input-width: 135px !default;
 
 	border: 0;
 	border-radius: 0;
-	border-bottom: $form-border-bottom;
-	box-shadow: none;
+	box-shadow: $form-shadow;
```
Comment (sgendre) [inline packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.scss]: once in our webapp, I get 
```
Undefined variable: "$form-shadow".
          in /home/me/workspace/data-streams/data-streams/webapp/node_modules/@talend/react-forms/lib/widgets/MultiSelectTagWidget/MultiSelectTagWidget.scss (line 44, column 14)
```

Comment (ybaskaran) [reply packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.scss]: the variable is declares in packages/theme/src/theme/_forms.scss

**Pull request merged**